src/styles/main.scss

```diff
@@ -1,3 +1,193 @@
-body {
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap');
+
+$color: #00acdc;
+$color-black: #060b35;
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  color: $color;
+}
+
+.header {
```

Comment:
according to BEM each block should be in its own file (header, nav, main, stars, card)